src/ray/util/logging.cc

```diff
@@ -367,7 +367,7 @@ void RayLog::InitLogFormat() {
     app_name_without_path = "DefaultApp";
   } else {
     // Find the app name without the path.
-    std::string app_file_name = ray::GetFileName(app_name);
+    std::string app_file_name = std::filesystem::path(app_name).filename().string();
```

Comment:
To reduce dependency for logging on filesystem, thus avoiding circular dependency.